Builder/src/KP2/House.java

```diff
@@ -0,0 +1,53 @@
+package KP2;
+
+public class House implements BuildingBuilder{
+
+	Building building;

```

Comment:
[High] Define this attribute as *private*. Also you can define it as *final* because you don't want that anyone change it once it is initialized.